redbot/cogs/mod/kickban.py

```diff
@@ -867,7 +867,6 @@ async def voiceban(self, ctx: commands.Context, user: discord.Member, *, reason:
     @checks.admin_or_permissions(ban_members=True)
     async def unban(self, ctx: commands.Context, user_id: RawUserIds, *, reason: str = None):
         """Unban a user from this server.
```

Comment:
```suggestion
        """Unban a user from this server.

```
Keep the space here, otherwise the next line will be bolded in the help command.